Introduction/README.md

```diff
@@ -224,59 +287,90 @@ Digest: sha256:8e03716b576e955c7606e4d8b8748c0f959a916ce16ba305ab262f042562340f
 Status: Downloaded newer image for fnproject/go:latest
  ---> 76aed4489768
 Step 8/10 : WORKDIR /function
- ---> Running in 619a412d5ab1
-Removing intermediate container 619a412d5ab1
- ---> 1487668015c4
+ ---> Running in 69ec68217d80
+Removing intermediate container 69ec68217d80
+ ---> 7dd3f73989ee
 Step 9/10 : COPY --from=build-stage /go/src/func/func /function/
- ---> cba454c3ad2c
+ ---> 17f42164b51f
 Step 10/10 : ENTRYPOINT ["./func"]
- ---> Running in 19e4eb479fc9
-Removing intermediate container 19e4eb479fc9
- ---> fc917eb83553
-Successfully built fc917eb83553
-Successfully tagged fndemouser/gofn:0.0.1
+ ---> Running in e2cee72aec64
+Removing intermediate container e2cee72aec64
+ ---> cde014cefdad
+Successfully built cde014cefdad
+Successfully tagged fndemouser/gofn:0.0.2
 
-{"message":"Hello World"}
+Updating function gofn using image fndemouser/gofn:0.0.2...
+Successfully created app:  goapp
+Successfully created function: gofn with fndemouser/gofn:0.0.2
+Successfully created trigger: gofn-trigger
 ```
 
-The last line of output `{"message":"Hello World"}` was produced
-by the Go statement `Msg: fmt.Sprintf("Hello %s", p.Name),`.
+All the steps to load the current language Docker image are displayed.
 
-Normally fn is run without the --verbose option. Let's rerun without the verbose output enabled.
+Functions are grouped into applications so by specifying `--app goapp`
+we're implicitly creating the application "goapp" and associating our
+function with it.
 
-![User Input Icon](images/userinput.png)
+Specifying `--local` does the deployment to the local server but does
+not push the function image to a Docker registry--which would be necessary if
+we were deploying to a remote Fn server.
+
+The output message `Updating function gofn using image fndemouser/gofn:0.0.2...`
+let's us know that the function is packaged in the image
+`fndemouser/gofn:0.0.2`.
+
+Note that the containing folder name `gofn` was used as the name of the
+generated Docker container and used as the name of the function that container
+was bound to. By convention it is also used to create the trigger name
+`gofn-trigger`.
+
+Normally you deploy an application without the `--verbose` option. If you rerun the command a new image and version is created and loaded.
+
+
+## Invoke your Deployed Function
+
+There are two ways to call your deployed function.  
+
+### Invoke with the CLI
+
+The first is using the `fn` CLI which makes invoking your function relatively
```

Comment:
Don't think `fn` here should be backquotes - should be "Fn CLI"

Reply:
Right. Fixed.